rustls/src/quic.rs

```diff
@@ -219,6 +219,11 @@ mod connection {
         pub fn is_early_data_accepted(&self) -> bool {
             self.inner.core.is_early_data_accepted()
         }
+
+        /// Returns the number of TLS1.3 tickets that have been received.
+        pub fn tls13_session_tickets_received(&self) -> u32 {
```

Comment:
Nit: let's drop the `session_` segment of this, similar to the other method and the inner API?

Reply:
:+1: